.github/workflows/ci.yml

```diff
@@ -58,6 +58,8 @@ jobs:
           - "7.3"
           - "7.4"
           - "8.0"
+          - "8.1"
+          - "8.2"
```

Comment:
Not sure if you already know this but there is a warning in logs `Deprecated: aws_crt_sign_request_aws(): Passing null to parameter #4 ($user_data) of type int is deprecated in /home/runner/work/aws-crt-php/aws-crt-php/src/AWS/CRT/Internal/Extension.php on line 27` which we probably should fix.

Reply:
I saw it, but didn't dig into it. Turns out it's a mismatch of arg type.